src/data/ACTIONS/root/RDM.ts

```diff
@@ -366,7 +366,7 @@ export const RDM = ensureActions({
 		speedAttribute: Attribute.SPELL_SPEED,
 		cooldown: 2500,
 		castTime: 5000,
-		potency: 420, // Increase White by 6
+		potency: 440, // Increase White by 6
```

Comment:
Potency changes should also be done in the layer file, to maintain consistency with other jobs.

Reply:
Same question as before, that wasn't something when I made changes for 7.01; there's also no pins in the discord describing what you're referring to either.